### DISCUSSION TITLE
Introduction Hello Everyone 🤠😁

### DISCUSSION THREAD

**Discussion by GitHubUser8196** [community/community#124534] [New to GitHub] [answered]
ANY TIPS FOR ME 😉😄

**Comment by davevad93** [comment]
Hi @Saisha16 ,

First thing I'd suggest you to make your profile's README and then [explore repos](https://github.com/explore), there are a lot of interesting things out there.
Regarding the README here's a sample template to start with:

```
## About Me
Hi everyone I'm [Your name]. I'm a [Field or Specialization]. 

## My Projects
- 🔭 I’m currently working on **[Project Name]**

- 🌱 I’m currently learning **[Something]**

- 👯 I’m looking to collaborate on **[Project Name]**

- 🤝 I’m looking for help with **[Project Name or something]**

- 💬 Ask me about **[Programming languages or frameworks you know]**

## Skills
![JavaScript](https://img.shields.io/badge/-JavaScript-F7DF1E?style=flat-square&logo=javascript&logoColor=black)
![Python](https://img.shields.io/badge/python-3670A0?style=for-the-badge&logo=python&logoColor=ffdd54)
![Linux](https://img.shields.io/badge/linux-3670A0?style=for-the-badge&logo=linux&logoColor=ffdd54)

## Contact Me
- Follow me on [Twitter](your_twitter)
- Send me an [email](mailto:your_email)
```
There are also tools that can help you make an awesome README, here's a list:
1. [GitHub Profile Readme Generator](https://rahuldkjain.github.io/gh-profile-readme-generator/) *with this you can generate very easily a good README to start with.*
2. [github-profile-trophy](https://github.com/ryo-ma/github-profile-trophy) *with this you can add dynamically generated GitHub Stat Trophies on your README.*
3. [github-readme-stats](https://github.com/anuraghazra/github-readme-stats) *with this you can dynamically generate GitHub stats on your README.*
4. [github-profile-header-generator](https://leviarista.github.io/github-profile-header-generator/) *with this you can generate a header for your profile, it's easy and straightforward to use.*
5. [github-stats-terminal-style](https://github.com/yogeshwaran01/github-stats-terminal-style) *with this you can dynamically generate GitHub stats looking like a Terminal Interface.*
6. [github-readme-activity-graph](https://github.com/Ashutosh00710/github-readme-activity-graph) *with this you can dynamically generate an activity graph to show your GitHub activities of last 31 days.*
7. [markdown-badges](https://github.com/Ileriayo/markdown-badges) *with this you can add badges to your Profile and Projects.*

**Reply by GitHubUser8196** [reply]
Actually other guy is asking for marked answer so.....

**Comment by Atharvkote** [comment, marked as answer]
Hi @Saisha16 
## Tips From my Side :
1.Create a repo with your username thts visible in your profile 
2.Design it there are several tutorial videos avaible on youtube you can refer that
3.add some projects if you have 
4. Its too early for you but still You can contribute to anyopen source available on github
5. You hunting github badge down for achievement purpose

### That its Wishing you best for your journey here on github❤️

Hi @Saisha16 if your free mark this question as answered if your are satisfied 🙂😁

**Reply by GitHubUser8196** [reply]
👍 Thanks
